src/layman/geoserver_proxy_test.py

```diff
@@ -401,4 +401,16 @@ def test_wfs_bbox():
 
     assert_all_sources_bbox(workspace, layer, expected_bbox)
 
+    data_xml = data_wfs.get_wfs20_delete_point(workspace, layer, )
```

Comment:
Let's make it in cycle with previous WFS-T with list of WFS-T XMLs and expected bboxes.